react-bootstrap/react-bootstrap.d.ts

```diff
@@ -943,8 +943,8 @@ declare module "react-bootstrap" {
         Addon: typeof InputGroupAddon;
         Button: typeof InputGroupButton;
     }
-    type InputGroup = React.ClassicComponent<InputGroupProps, {}>;
-    var InputGroup: React.ClassicComponentClass<InputGroupProps>;
+    type InputGroup = React.Component<InputGroupProps, {}>;
+    var InputGroup: InputGroupClass;
```

Comment:
Good to know. But im just fixup this little part wich was not aligned with other components to be able to use <InputGroup.Addon> in ts. If there is something wrong new issue with proposed changes should be made.